toolkit/tools/internal/timestampcsvparser/timestampcsvparser.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Parses the timestamps stored in CSV files and print them
+// to the terminal at the end of the build.
+
+package timestampcsvparser
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+var (
+	timeArray [][]string // A 2-D array used to store all timestamps from CSVs.
+	// A list of CSV files to be parsed.
+	files []string
+)
+
+// Reads a CSV file and appends line by line to array.
+func CSVToArray(filename string) {
+	file, err := os.Open(filename)
+
+	if err != nil {
+		fmt.Println("failed to open CSV file")
```

Comment:
1. Let's use a logger instead of printing directly to `stdout`.
2. We cannot continue in case of a failure here and should error-out.

Reply:
Please replace other `fmt.Print*` calls with a logger.

Reply:
Will address via switch to .json format.